### ISSUE TITLE
Incorrect stock splits for certain tickers

### ISSUE THREAD

**Issue by Newtoniano** [polygon-io/issues#311]
While doing sanity checks on the data from the `/v3/reference/splits` endpoint, with the following snippet I find 216 splits with the same `ticker` and `execution_date`. One of these multiple entries is definitely wrong and the adjusted ohlcv data that results from applying them sequentially (both manually on my end or from your aggregates endpoint) is totally off.

To identify the splits with multiple entries using your python client you can run this in a jupyter notebook:
```
from polygon import RESTClient
import pandas as pd

client = RESTClient(
    polygon_api_key
)
today = pd.Timestamp("today").strftime("%Y-%m-%d")

splits = []
for split in client.list_splits(limit=1000, execution_date_lte=today):
    splits.append(split)

splits_df = pd.DataFrame(splits).sort_values("execution_date").reset_index(drop=True)
splits_duplicates = splits_df[
    splits_df.duplicated(subset=["ticker", "execution_date"], keep=False)
]
print(f"Found {len(splits_duplicates)} splits with the same ticker and execution_date")
splits_duplicates.tail(20)
```

Let's consider the splits for these two tickers for example:
![image](https://github.com/user-attachments/assets/a336ae9e-4d0d-4230-a3ca-6626beda24ac)

For `SOPA`, there has been only the 15-1 reverse stock split on 2024-05-01, not sure why there is a second entry. If you apply just the 15-1 split, by multiplying all the unadjusted ohlc prices before 2024-05-01 by 15 you get to the correct adjusted ohlcv prices, if you apply both sequentially, obviously everything is off. Unadjusted data is:
![image](https://github.com/user-attachments/assets/1071b7e8-85cb-494d-b2da-2c682c9ac640)
But your adjusted endpoint shows 
![image](https://github.com/user-attachments/assets/6903d66a-b2cf-45da-92d2-ca46b324c572)
since it's also dividing by 2 because of your additional (incorrect) split data entry.

Regarding `SCCO`, the split should be just the 1.0104 entry, but there is another one, and the adjusted ohlcv series is of course majorly off:
![image](https://github.com/user-attachments/assets/a3d8697a-4a36-4d26-bf1d-d6b5e57c3fdf)

The bottom line is that there are hundreds of split entries in your records that should be removed entirely/corrected, namely one of the split entries for the same day and same ticker. All the downstream calculations such as adjusted ohclv aggregates should work just fine without any further modification once the split data is fixed.

**Comment by AHangstefer** [comment]
Hi! Thanks for reaching out. The duplicate SCCO and SOPA splits have been removed. If you're able to share a list of symbol that have this issue, I'd be happy to take a look and make sure they get corrected.
